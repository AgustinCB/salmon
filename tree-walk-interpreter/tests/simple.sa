print "HOLA";
var a=nil;
print a;
print a=2;
print a+4;