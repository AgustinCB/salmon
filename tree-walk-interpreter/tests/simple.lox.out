HOLA
Nil
2
6
